src/quo2/components/dividers/strength_divider/view.cljs

```diff
@@ -0,0 +1,85 @@
+(ns quo2.components.dividers.strength-divider.view
+  (:require [react-native.core :as rn]
+            [quo2.components.dividers.strength-divider.style :as style]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [react-native.linear-gradient :as linear-gradient]
+            [react-native.svg :as svg]
+            [quo2.components.icon :as icon]))
+
+(def status-list
+  {:very-weak   {:default-text "Very weak"
```

Comment:
since these are in the components they should be i18n labels. You can add the labels to `en.json` and then use the method `(i18n/label :t/<your-key>)`